public/app/features/alerting/unified/utils/url.ts

```diff
@@ -1,10 +1,14 @@
 import { config } from '@grafana/runtime';
 
-export function createUrl(path: string, queryParams?: string[][] | Record<string, string> | string | URLSearchParams) {
+export function createRelativeUrl(
+  path: string,
+  queryParams?: string[][] | Record<string, string> | string | URLSearchParams
+) {
   const searchParams = new URLSearchParams(queryParams);
   const searchParamsString = searchParams.toString();
+  const pathWithSlash = path.startsWith('/') ? path : `/${path}`;
 
-  return `${config.appSubUrl}${path}${searchParamsString ? `?${searchParamsString}` : ''}`;
+  return `${config.appSubUrl}${pathWithSlash}${searchParamsString ? `?${searchParamsString}` : ''}`;
 }
 
 export function createAbsoluteUrl(
```

Comment:
Any particular reason why we aren't also checking for leading `/` when passing path in to `createAbsoluteUrl`?

Reply:
no, let's do it!